### PR TITLE
v2 - Prove of Concept

### PR DESCRIPTION
**Pull request by daKmoR**
Shows the overall idea

- you can index your packages (if they have a `custom-elements.json`) and then you can search
- show possible multiple filters
- shows mobile behaviour for filters, list, detail view
- shows details for readme, demo, links, source
- only the dependencies filters are implemented
- in alpha you need to manually index via `./add.html`.
- demos are an `demo/index.html` or any url specified in package.json as (demoUrl)
- demos are served directly from unpkg no code changes are done (e.g. it will need to run natively)

=> styling is basically non existing

See it in action
https://deploy-preview-7--catalog-open-wc-org.netlify.com/

See all the parts
https://deploy-preview-7--catalog-open-wc-org.netlify.com/storybook

### CODE REVIEW

File: packages/search-item/src/owc-search-item.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+          <a href="#"><img src="https://img.shields.io/badge/lib-haunted--4.x-blueviolet.svg"/></a>
+        `;
+      default:
+        return html``;
+    }
+  }
+
+  render() {
+    return html`
+      <div id="name">
```
Comment (bennypowers) [inline packages/search-item/src/owc-search-item.js]:
```suggestion
      <header id="name">
```

File: packages/search-item/src/owc-search-item.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+          <h1>
+            <a href="#details" target="_blank">${this.name}</a>
+          </h1>
+          ${this.renderRegisteredTypes()}
+        </div>
+
+        <div>
+          ${this.description}
+        </div>
+      </div>
```
Comment (bennypowers) [inline packages/search-item/src/owc-search-item.js]:
```suggestion
      </header>
```

File: packages/search-item/src/owc-search-item.js (hunk 1 above)
Comment (bennypowers) [reply packages/search-item/src/owc-search-item.js]: consider also wrapping the whole tree in an `<article>`

Comment (daKmoR) [reply packages/search-item/src/owc-search-item.js]: got changed quite a lot 🙈  please revisit if needed 👍

**Pull request merged**